terraform/gcp/projects/cluster.tfvars.template

```diff
@@ -74,15 +81,14 @@ notebook_nodes = {
   }
 }
 
-# Setup a single node pool for dask workers.
-#
-# A not yet fully established policy is being developed about using a single
-# node pool, see https://github.com/2i2c-org/infrastructure/issues/2687.
-#
+{% if hub_type == "daskhub" %}
 dask_nodes = {
+#  A not yet fully established policy is being developed about using a single
+#  node pool, see https://github.com/2i2c-org/infrastructure/issues/2687.
   "n2-highmem-16" : {
     min : 0,
     max : 200,
     machine_type : "n2-highmem-16",
   },
 }
+{% endif %}
```

Comment:
If we make this commented out instead, do we then need to provide hub_type still in this context?

Reply:
Yes, I think so, but we'll also need to comment it out in the aws template https://github.com/2i2c-org/infrastructure/blob/d8018e18e41fadd2b9956934ca3cfbc1c1f2055b/eksctl/template.jsonnet#L43-L59.

But I think it's worth keeping with the change in this PR, where you just need to hit enter when you don't know the hub type:
<img width="1440" alt="Screenshot 2024-04-03 at 16 52 36" src="https://github.com/2i2c-org/infrastructure/assets/7579677/4c8aa560-b2e0-44db-85b8-7557b057d696">

I also like the advantages that you get if you know from the beginning there will be daskhubs, i.e. don't change and regenerate afterwards. I think it's a decent code complexity compromise. Do you agree?

Reply:
Absolutely, lets go! Thanks for reflecting on this!